RELEASE.md

```diff
@@ -63,6 +63,7 @@
 
 *   XLA:
     *   MWMS is now compilable with XLA.
+    *   Compute Library for the Arm® Architecture (ACL) is supported for aarch64 cpu xla runtime
```

Comment:
I added this change to #57402 instead to avoid merge conflicts. (Otherwise, PRs can't be merged successively and we will have to ask you to resolve merge conflicts once #57402 is in.) Could you please remove changes to this file from this PR? If you'd like to make further changes to RELEASE.md, please suggest changes to #57402.

Reply:
sounds good. thank you!